test/sync/src/commonMain/kotlin/io/realm/test/mongodb/TestApp.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("invisible_member", "invisible_reference")
+
+package io.realm.test.mongodb
+
+import io.ktor.client.request.get
+import io.realm.internal.platform.runBlocking
+import io.realm.internal.platform.singleThreadDispatcher
+import io.realm.mongodb.App
+import io.realm.mongodb.appConfigurationOf
+import io.realm.test.mongodb.util.AdminApi
+import io.realm.test.mongodb.util.AdminApiImpl
+import io.realm.test.mongodb.util.defaultClient
+import kotlinx.coroutines.CoroutineDispatcher
+
+const val BASE_CMD_SRV_URL = "http://127.0.0.1:8888"
```

Comment:
I don't think that letters are so expensive that we cannot call this `BASE_COMMAND_SERVER_URL`?  ;)